gc-cli/bin/gc-cli.js

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env node
 "use strict";
 
+require("babel-register");
```

Comment:
sorry Brian I thought I already commented on this. I don't fully understand why but when I added `ignore: false` the script does not work.